packages/icons/icons.json

```
@@ -3883,5 +3883,12 @@
         "tags": "open, application, new-tab, external",
         "group": "interface",
         "codepoint": 62251
+    },
+    {
+        "displayName": "Float",
+        "iconName": "float",
+        "tags": "number, float, double, decimal",
+        "group": "interface",
```

let's put it into Miscellaneous with the other (array) data types. the icons list is due for a re-organization some time; we should have a separate category for data types...

```suggestion
        "group": "miscellaneous",
```